tools/macos_buildenv.sh

```diff
@@ -78,12 +78,39 @@ case "$COMMAND" in
             echo "Build environment found: ${BUILDENV_PATH}"
         fi
 
+        export SDKROOT="${BUILDENV_BASEPATH}/MacOSX10.13.sdk"
```

Comment:
Shall we define a target SDK that is by default set to the current SDK? Then we only need to go the extra mile of downloading it and manipulating env variables if it doesn't match the native SDK.

Reply:
No, practically nobody is using a version of XCode that has such an old macOS SDK anymore.